### PR TITLE
Move all the good stuff from v0.5.2 to master [#1693]

### PR DESCRIPTION
**Pull request by jbuck**

### CODE REVIEW

**Comment by dseif** [comment]
mother of god

**Comment by cadecairos** [comment]
0.0

**Comment by benrito** [comment]
Heroic patch.

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @humphd received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/0d4c9c7257cbef4/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/0d4c9c7257cbef4/output.txt

Total script time: 0.36 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @humphd received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/97d42b787a01a45/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/97d42b787a01a45/output.txt

Total script time: 0.89 mins

Server is running on http://pr617.butyr.org/templates/test.html

File: src/timeline/status.js [hunk 1]
```diff
@@ -20,20 +20,20 @@ define( [], function(){
...
 
     function onMouseUp( e ){
-      _button.removeAttribute( "data-mouse-state" );
+      _button.removeAttribute( "mouse-state" );
       window.removeEventListener( "mouseup", onMouseUp, false );
     } //onMouseUp
 
     _button.addEventListener( "mousedown", function( e ){
-      _button.setAttribute( "data-mouse-state", "depressed" );
+      _button.setAttribute( "mouse-state", "depressed" );
```
Comment (secretrobotron) [inline src/timeline/status.js]: All of the changes to how we apply/interact with `data-*` attributes look like they have the reverse change applied to them. They _should_ all be `data-*`, and we had a ticket to change them to be appropriate. This oddity caught my eye in the CSS change above too -- the same switch happened. Perhaps this change didn't get passed back to 0.5.x properly.

**Pull request merged**